odl/test/solvers/nonsmooth/admm_test.py

```diff
@@ -72,5 +74,65 @@ def test_admm_lin_l1():
     assert all_almost_equal(x, data_1, places=2)
 
 
+def test_admm_lin_vs_simple():
```

Comment:
I guess these are old, or do we intend for them to stay? If so, I'd move the "simple" guy into this file